indexing-service/src/main/java/io/druid/indexing/worker/WorkerCuratorCoordinator.java

```diff
@@ -127,26 +128,13 @@ public void stop() throws Exception
 
   public void makePathIfNotExisting(String path, CreateMode mode, Object data) throws Exception
```

Comment:
do we still need this method? Might as well call `CuratorUtils.createIfNotExists` directly